src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPageVBWinForms.vb

```diff
@@ -991,6 +992,8 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
             PopulateTargetFrameworkComboBox(TargetFrameworkComboBox)
 
+            ' Hide the AssemblyInformation button if project supports Pack capability, and hence has a Package property page with assembly info properties.
+            EnableControl(AssemblyInfoButton, ProjectHierarchy.IsCapabilityMatch(Pack))
```

Comment:
Oops! Yes good catch.  I changed this last minute from just directly disabling to calling EnableControl and messed it up. Fixed now.